templates/user/auth/captcha.tmpl

```diff
@@ -1,11 +1,10 @@
 {{if .EnableCaptcha}}{{if eq .CaptchaType "image"}}
 	<div class="inline field">
-		<label>{{/* This is CAPTCHA field */}}</label>
 		{{.Captcha.CreateHTML}}
 	</div>
 	<div class="required inline field {{if .Err_Captcha}}error{{end}}">
 		<label for="captcha">{{ctx.Locale.Tr "captcha"}}</label>
-		<input id="captcha" name="captcha" value="{{.captcha}}" autocomplete="off">
+		<input id="captcha" class="gt-w-full" name="captcha" value="{{.captcha}}" autocomplete="off">
```

Comment:
I just realized that it is playing a CSS style overriding game ....

For example: by Fomantic UI: the `inline field` above means that the label is the same row as the input

https://fomantic-ui.com/collections/form.html#inline-field


So, by default, without the `inline`, it should be the UI you need. 


I do not think it is right to use "inline" outside, and change the "inline" behavior inside.

There are already many UI bugs caused by such unclear overridings.

Reply:
I understand this perspective. I removed the inline class to allow the default behavior of the `div` to take over. 

On the topic of helper classes, we recently implemented tailwinds. We should update our contribution docs to reflect the usage of these classes? If tailwinds is the path forward, I'll open a PR to update these docs.

https://github.com/go-gitea/gitea/blob/4b8293aa094e725b372329a19da687a6d1550069/docs/content/contributing/guidelines-frontend.en-us.md?plain=1#L50

Reply:
I think it should be updated accordingly. At the moment the plan is to remove most Gitea's own helpers, use Tailwind as much as possible.